src/test/java/api/support/fixtures/ScheduledNoticeProcessingTimerClient.java

```diff
@@ -0,0 +1,25 @@
+package api.support.fixtures;
+
+import static api.support.RestAssuredClient.post;
+
+import java.net.URL;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeUtils;
+
+import api.support.APITestContext;
+import io.vertx.core.json.JsonObject;
+
+public class ScheduledNoticeProcessingTimerClient {
+
+  public void runNoticesProcessing(DateTime mockSystemTime) {
+    DateTimeUtils.setCurrentMillisFixed(mockSystemTime.getMillis());
+    runNoticesProcessing();
+    DateTimeUtils.setCurrentMillisSystem();
+  }
+
+  public void runNoticesProcessing() {
+    URL url = APITestContext.circulationModuleUrl("/circulation/scheduled-notices-processing");
+    post(new JsonObject(), url, 204, "scheduled-notices-processing-request");
```

Comment:
Does Okapi send an empty JSON object when posting to this endpoint?

Reply:
Okapi documentation says it sends empty body. As there was no appropriate overload of `post()`, I used empty json object.